src/common/git/husky/assets/check-commit-msg.js

```diff
@@ -7,8 +7,15 @@ function debug(debugMessage) {
 }
 
 const pattern = /^\w{2,4}-\d+\s/
-const message = process.argv[2]
-debug(`Test message "${message}" with pattern "${pattern}"`)
+const [message, currentBranch] = process.argv.slice(2)
+const ignoreBranches = '{{BRANCHES}}'.split(',').filter(Boolean)
```

Comment:
The `.filter(Boolean)` part ensure an empty string is not parsed into `''`. Moreover we don't need to handle accidental empty string inputs.